README.md

```diff
@@ -1,11 +1,20 @@
 # nfs
 
+[![Build Status](https://github.com/voxpupuli/puppet-nfs/workflows/CI/badge.svg)](https://github.com/voxpupuli/puppet-nfs/actions?query=workflow%3ACI)
```

Comment:
can you pull the README.md cleanup into it's own PR? It's not really related to the puppet 8 implementation